sdk/cosmos/.azure-pipelines/cosmos.test.nightly-emulator.yml

```diff
@@ -0,0 +1,47 @@
+jobs:
+  - job: 'Test'
+
+    timeoutInMinutes: 120
+    continueOnError: false
+    strategy:
+      matrix:
+        Windows_Python37:
+          OSVmImage: 'vs2017-win2016'
+          PythonVersion: '3.7'
+      maxParallel: 1
```

Comment:
You should remove the `maxParallel` setting.  It currently doesn't matter since the matrix only has a single row, however if and when we add more rows, we **do** want them to run in parallel to decrease the total time.  And since these tests use the emulator instead of a live Azure endpoint, there should be no issues running in parallel.